manifests/init.pp

```diff
@@ -45,6 +45,10 @@
   String[1]            $crontab_mailto          = 'root',
   Optional[Stdlib::Absolutepath] $crontab_home  = undef,
   Cron::Run_parts      $crontab_run_parts       = {},
+  Cron::User           $crontab_owner           = 'root',
+  String[1]            $crontab_group           = '0',
```

Comment:
I used 'root' for the group but the spec tests failed because the tests are looking for '0' as the group.  I was wondering why the tests used '0' instead of 'root' but I have no experience with rspec tests so I just failed back to '0'.  I'm happy to switch the group to 'root' and update the merge request but you may experience a lot of merge noise due to my ignorance on spec test writing.

Reply:
Using '0' for the group was a hackish way of not shocking on the name of the group with gid 0 which happen to be 'root' on Linux, 'wheel' on FreeBSD and has probably another name on a few other Operating Systems.

Adjusting the value accordingly in the test suite should be fine: default to 'root', adjust the test suite to check that the value is 'root', and add a `data/os/FreeBSH.yaml` file with:

```yaml
---
cron::crontab_group: wheel
```